.dagger/test.go

```diff
@@ -350,7 +353,9 @@ func (t *Test) testCmd(ctx context.Context) (*dagger.Container, error) {
 		WithMountedFile(cliBinPath, devBinary).
 		WithEnvVariable("_EXPERIMENTAL_DAGGER_CLI_BIN", cliBinPath).
 		WithEnvVariable("_EXPERIMENTAL_DAGGER_RUNNER_HOST", endpoint).
-		With(t.Dagger.withDockerCfg) // this avoids rate limiting in our ci tests
+		With(t.Dagger.withDockerCfg). // this avoids rate limiting in our ci tests
+		WithExec([]string{"sh", "-c", "go install gotest.tools/gotestsum@v1.12.0"})
```

Comment:
We should avoid installs at this point - this will require a fresh download every single time we change any of the things in the pipeline above.